apps/demo/app/routes/index.tsx

```diff
@@ -91,6 +98,15 @@ export default function Index() {
               onClick: toggleMode,
               icon: (mode ? DarkMode : LightMode) as IconType,
             },
+            {
+              'aria-label': theme
+                ? 'Switch to New Theme'
+                : 'Switch to Default Theme',
+              title: theme ? 'Switch to New Theme' : 'Switch to Default Theme',
+              as: 'button',
+              onClick: toggleTheme,
+              icon: (theme ? ArrowLeft : ArrowRight) as IconType,
```

Comment:
I don't know if I like the arrow here, not totally clear what it's doing. Maybe a palette, or paint brush icon would be better?